deployment/helm/resource-management-policies/balloons/templates/configmap.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nri-resource-policy-config.default
+  namespace: {{ .Values.namespace }}
```

See https://github.com/containers/nri-plugins/pull/45#discussion_r1227963129

```suggestion
  namespace: {{ .Release.namespace }}
```

Applies to all templates